tests/cypress/integration/actions_tasks_objects/case_13_merge_split_features.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Merge/split features', () => {
+    const caseId = '13';
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        switchLabel: false,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+    const createRectangleShape2PointsSecond = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        switchLabel: false,
+        firstX: createRectangleShape2Points.firstX + 300,
+        firstY: createRectangleShape2Points.firstY,
+        secondX: createRectangleShape2Points.secondX + 300,
+        secondY: createRectangleShape2Points.secondY,
+    };
+    const frameNum = 0;
+    // Check the 'X' coordinate. 'Y' coordinate is the same.
+    let xCoordinatesObjectFirstFrame = 0;
+    let xCoordinatesObjectThirdFrame = 0;
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    function checkFrameNumber(frameNum) {
+        cy.get('.cvat-player-frame-selector').within(() => {
+            cy.get('input[role="spinbutton"]').should('have.value', frameNum);
+        });
+    }
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create rectangle shape on first frame', () => {
+            checkFrameNumber(frameNum);
+            cy.createRectangle(createRectangleShape2Points);
+            cy.get('#cvat_canvas_shape_1').should('have.attr', 'x').then(xCoords => {
+                xCoordinatesObjectFirstFrame = Math.floor(xCoords);
+            });
+        });
+        it('Create rectangle shape on third frame with another position', () => {
+            cy.get('.cvat-player-next-button').click().click();
```

Comment:
Just to the future. You can input specify a frame in ``.cvat-player-frame-selector`` and to not click these buttons several times.

Reply:
Redesigned the function to enter the required frame number.